### DISCUSSION TITLE
Error: creating EC2 Subnet: InvalidSubnet.Conflict: The CIDR '10.0.3.0/24' conflicts with another subnet

### DISCUSSION THREAD

**Discussion by gmonika136** [learndevopsonline/learndevopsonline#1513] [Q&A]
Hi 

Still i am getting same error even though i changed public & private subnet ip's.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129938676/5a19628b-f547-41a9-a0b5-96b2916a8d43)

**Comment by gmonika136** [comment]
To change the private & public are same before

**Reply by sreedharm07** [reply]
resolved in lab query session.
